### ISSUE TITLE
integer overflow in luaotfload-harf-var-tttf.lua

### ISSUE THREAD

**Issue by marcpenninga** [latex3/luaotfload#221]
I get this error (luaotfload-harf-var-ttf.lua version 3.21, 2022-03-18, in combination with lualatex 1.15.0 from TL2022-pretest):
/home/marc/texlive/2022/texmf-dist/tex/luatex/luaotfload/luaotfload-harf-var-ttf.lua:332: bad argument #4 to 'pack' (integer overflow).

Minimal example:
```
\documentclass{article}
\font\testfont = [./AndadaPro[wght].ttf]:mode=harf,axis={wght=410}
\begin{document}
\testfont \'{a}
\end{document}
```
(Using the free font from https://github.com/huertatipografica/Andada-Pro/blob/master/fonts/variable/AndadaPro%5Bwght%5D.ttf). 

Three final remarks (don't know if relevant or not):
- The code works with lualatex from TL2021 (1.13.2).
- The error only happens for weights >= 410; with smaller weights, the code compiles fine.
- Although AndadaPro is a .ttf font, it has 1000 UnitsPerEm instead of the more common 2048.

**Comment by zauguin** [comment]
Thanks for the report.

> * The code works with lualatex from TL2021 (1.13.2).

In TL2021 `harf` mode doesn't support variable fonts, so there the `axis` argument is just ignored. 

Anyway it should be fixed in the `dev` branch with [4d0765d](https://github.com/latex3/luaotfload/commit/4d0765db381333d263bab2fdeb233b7c826e8a78) and it would be great if you could test the fix.

**Comment by marcpenninga** [comment]
Thanks for the quick fix! I can confirm that this fixes the issue.
